.github/workflows/conformance-clustermesh.yaml

```diff
@@ -272,10 +272,14 @@ jobs:
           fi
 
           CILIUM_INSTALL_INGRESS=""
+          # Once https://github.com/cilium/cilium/issues/31653 is fixed, we can remove tunnel check
           if [ "${{ matrix.kube-proxy }}" == "none" ]; then
-            # The ingress controller requires KPR to be enabled.
-            CILIUM_INSTALL_INGRESS="--helm-set=ingressController.enabled=true \
-              --helm-set-string=kubeProxyReplacement=true"
+            CILIUM_INSTALL_INGRESS="--helm-set=ingressController.enabled=true"
+            # Use the legacy host routing in case of tunnel disabled
+            # Related to https://github.com/cilium/cilium/pull/22006
```

Comment:
This makes me think that Cilium's `devices` option might not be set up correctly for this test ...

Reply:
What do you mean? Devices are not explicitly configured in this workflow, and should be autodetected. But there should be only one interface (unless something should be attached to loopback as well).

Reply:
There's probably not enough context in why we point to #22006. But that PR usually has been problematic before (https://github.com/cilium/cilium/pull/28264) because it potentially short-cuts the BPF-redirect, if Cilium manages only one external interface.

Do we have a good understanding why BPF Host-Routing is no good for the Ingress scenario - and why it only affects native routing?

Reply:
thanks for your comment, we will continue the investigation in #31653, legacy host routing can be considered as a workaround.